pypeclub/OpenPype
# Ftrack: Add description integrator

## Brief description
Added integrator which changes description of integrated AssetVersion.

## Description
Integrator works similar way like integrate ftrack note but text is stored to AssetVersion description instead of note. It is possible to define template where `{comment}` and `{intent}` can be used. The plugin is disabled by default to not duplicate text in comment and note.

Added collector which collects intent label logic which was in Integrate Ftrack Note plugin.

## Testing notes:
1. Enable the plugin
2. Publish something that will be published to ftrack
3. Check description of new asset versions

Depends on https://github.com/pypeclub/OpenPype/pull/3025
- implementation is based on changes in the PR
